src/zcash/Address.cpp

```diff
@@ -67,6 +67,12 @@ SaplingIncomingViewingKey SaplingFullViewingKey::in_viewing_key() const {
     return SaplingIncomingViewingKey(ivk);
 }
 
+uint256 SaplingFullViewingKey::GetHash() const {
+    CDataStream ss(SER_NETWORK, PROTOCOL_VERSION);
+    ss << *this;
+    return Hash(ss.begin(), ss.end());
```

Comment:
We should use the fingerprint defined in ZIP 32, for consistency: `BLAKE2b-256("ZcashSaplingFVFP", fvk)`

```cpp
CBLAKE2bWriter ss(SER_GETHASH, 0, ZCASH_SAPLING_FVFP_PERSONALIZATION);
ss << *this;
return ss.GetHash();
```